Add stakeeasy and update polar description (#18)
SecretFoundation/awesome-secret: README.md

    * **Silk** - Algorithmic stablecoin with transactional privacy by default
    * **Synthetics** - Algorithmic synthetic assets with transactional privacy by default
    * Many more will be revealed :shushing_face:

> * :soon: [StakeEasy](https://stakeeasy.finance/) - Privacy preserving liquid staking solution for secret network.

Reviewer: ```suggestion
* :soon: [StakeEasy](https://stakeeasy.finance/) - Privacy preserving liquid staking solution for Secret Network.
```

* [SecretJS](https://www.npmjs.com/package/secretjs) - JavaScript library for interacting with the network
    ([examples](https://github.com/enigmampc/SecretJS-Templates))
* [Griptape.js](https://griptapejs.com/) - By Validator [Stake or Die][stakeordie], a front-end web framework for developing DApps on the network

> * [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a framework for building secret contracts. It features facilities for [project templates](https://github.com/arufa-research/polar-templates), compilation, deployment, javascript interaction (based on schemas), and testing

Author: ```suggestion
* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a framework for building secret contracts. It features facilitates for [project templates](https://github.com/arufa-research/polar-templates), compilation, deployment, javascript interaction (based on schemas), and testing
```

> * [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a development framework for building secret contracts. It facilitates features such as initiating project repo from [contract templates](https://github.com/arufa-research/polar-templates), easy compilation of contracts, deployment, Interacting with contracts using schema and contract testing framework.

Reviewer: This line should ideally be an "executive summary" of the project, and ideally shouldn't be longer than one line when rendered in the main page. That being said, the additional information is useful. Besides that, you have a couple minor typos here.

So, I think we can either just fix the typos:
```suggestion
* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a development framework for building secret contracts. It facilitates features such as initializing project repos from [contract templates](https://github.com/arufa-research/polar-templates), easy compilation of contracts, deployment, Interacting with contracts using their schema, and a contract testing framework.
```

Or at least condense the information to make it somewhat shorter:
```suggestion
* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a framework for building secret contracts. It features facilities for [project templates](https://github.com/arufa-research/polar-templates), compilation, deployment, javascript interaction (based on schemas), and testing
```

Author: got it. will condense the information
